release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -2639,6 +2639,8 @@ link:https://access.redhat.com/solutions/6762051[{product-title} 4.9.23 containe
 
 * Previously, installation failed when installing to a region that had local zone enabled. With this update, the installation program considers only availability zones and not local zones. Now, installation with local zones enabled will only install to the availability zones in that region and not to any local zones. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2052307[BZ#*2052307*])
 
+* Previously, {product-title}, with OVN-Kubernetes, managed ingress access to services via ExternalIP. When upgrading from 4.8.33 to 4.8.34, access `ExternalIP` stops work with issues like "No Route to Host". With this update, administrators will now have to direct traffic from externalIPs to the cluster. For guidance, see (link:https://access.redhat.com/solutions/6954851[KCS*]) and (link:https://kubernetes.io/docs/concepts/services-networking/service/#external-ips[*Kubernetes External IPs*]) (link:https://bugzilla.redhat.com/show_bug.cgi?id=2076662[*BZ#2076662*])
```

Comment:
PTAL the versions and make sure they're correct here. This is for 4.9, but references 4.8 in the text: 

* Previously, OVN-Kubernetes managed Ingress access to services through ExternalIPs. When upgrading from 4.8.33 to 4.8.34, accessing ExternalIPs stopped processes and returned errors like `No Route to Host`. With this update, administrators will now have to direct traffic from ExternalIPs to the cluster. For guidance, see (link:https://access.redhat.com/solutions/6954851[KCS*]) and (link:https://kubernetes.io/docs/concepts/services-networking/service/#external-ips[*Kubernetes External IPs*]) (link:https://bugzilla.redhat.com/show_bug.cgi?id=2076662[*BZ#2076662*])